mageia/README.md

```diff
@@ -9,7 +9,7 @@ For detailed information about the virtual/transfer sizes and individual layers
 
 # What is Mageia?
 
-[Mageia](http://www.mageia.org) is a GNU/Linux-based, Free Software operating system.It is a [community](https://www.mageia.org/en/community/) project, supported by [a nonprofit organisation](https://www.mageia.org/en/about/#mageia.org) of elected contributors.
+[Mageia](http://www.mageia.org) is a GNU/Linux-based, Free Software operating system. It is a [community](https://www.mageia.org/en/community/) project, supported by [a non-profit organisation](https://www.mageia.org/en/about/#mageia.org) of elected contributors.
```

Comment:
This needs to be done in the `content.md`; the bot will update `README.md` from there automatically.